edx_proctoring/backends/tests/test_software_secure.py

```diff
@@ -649,7 +649,7 @@ def test_review_on_archived_attempt(self):
         )
 
         # now delete the attempt, which puts it into the archive table
-        remove_exam_attempt(attempt_id)
+        remove_exam_attempt(attempt_id, "null_user")
```

Comment:
As discussed, you can add an assertion that what is passed is actually a user to show how this should be used.  If you could name the variable ``requested_user``, I think that would make this more clear.